src/Fantomas.Core/CodePrinter.fs

```diff
@@ -1401,10 +1401,17 @@ and genExpr astContext synExpr ctx =
             )
 
         | IndexWithoutDotExpr (identifierExpr, indexExpr) ->
-            genExpr astContext identifierExpr
-            +> sepOpenLFixed
-            +> genExpr astContext indexExpr
-            +> sepCloseLFixed
+            match indexExpr with
+            | Sequential _ ->
```

Comment:
I think it is *ok* to always have the sequential case when the expression is multiline.
This problem isn't necessarily related to `SynExpr.Sequential`, more general to multiline expressions. [Example](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKAbRAXMBDAMrEcSQHYDm%2BAlgM44C8YA2mAAxgDcYAjGALoAdAE4DSw0ZhwAzKkLph640hAhSArsoAeYALQA%2BJSqN5CJMxWp0Wh47YDKsALaIAwulw0aAOjtZcWKmgAWURHACNEIQAKG1s4sAB6BLByIngwOEdnUixY%2BONcIXJuABo8-JVC8gAmMpFlCuN1ZQBPXQN6xvyksHKulTaAah4%2B2wBKLwAxIlH%2BSBLwKBpsAIoaVGZFiFBbSAASLBaAB2Q0SCoc%2Ba2VfYA3XHQ1U8341gX4yABJUiQcuyoAF7Id5xAAs1wgfGuAF8QdsIVADsdnudLnCbnt7o9nkwERBSnioEFcJpqKREPhEBQsAALK7xbjVVgIqFGWHXHbGfaHE4bSBU%2BAAeSkZMQvhamHpXMxDyeG1x8VqhMgAFEfsLRVLdugpJAWTD0ZyjNzkXywkR0FqMVi5WgFXEAMzo43fZZCLCTC4PAByiAA7ugLsDCVghE99WzDQiTbyzubYJbnYibTjCaCkxBIHYjrhoIgAEKIKSwISIAAKhVwziwkStRlD4dsrJU7KMRoxPJR8cT0Zl2PlhIArBmoNncwWiyWKbA-ZFoB5EN9brB5wFYKQ6yoG4gIy2o7skbGoN3N3dZan4gA2EdZnN5wvF0sAVSOJyE8%2BWS5X-io683qgeZZdwgVsVHbTNDy7C1Tz7W0XjiAB2G8x3vSdSzcDwaBcdc6DDaAsBLf8wCkQCdybA0OV7TszWgpMz37O1CQADmQu8J0fRAQnCWsRxI9AgPIyNKIPai41o3sUwHeIAE5WPHB8p2w9A-140jgLAUD4RE00xITGDJMYhk3mVFDEAAQSkGshGwrJcCI7d1M0sBwMRUTj3Eg8DPg2xuAJD5TIU0s7FCQIExUwk%2BIE4xmxA-dpTcyATzo2CLziRk5LzCyrOCxxQuUjcRwcwS92E%2BKdPcvTkq8%2B0fKdEy2LMoRYA0eAABFEEDXKrPssMyOiii2yo8rUSwfTzyktL02VYlNE%2BKQABUaSpOwaRLLAAHUqHgWkiOZYrYtK41IL5C5Rqq8bDLS4dppJObFqpFV%2BLFVb3U27a6RHS89v6oTBu0o8RrGhjvOMbhrxu2bSBkTRBTffwSxVTQjlLTxfwK5jvqMGKNLio6EtOoG4Jq0GkIhgAlRA4CEeA3p2kdQUxlRsaclyYxRAnzuB4mjG4Fjycpkt4G9NRuKEYVPhrRx1hHbhHNxjthscNR0ACQNyWmIRHH8Kz5tNTmicJbhZOVCmqfgIJldVoMNa1rBupvaAaUrfDIgAfT9Ladrlw6FYBjmJIukGjCZG8ZsaoRcBaQUhEsDbPY%2BjHvb%2Bsq-bRAOucJao-LiSAw6ECOo5j2gsGF0Xxcl6XDaTsChoBpWVaoNXEBt7XIl13l9dS2xqiVD5w8j6PY4thum5bu2eOVR3nas93471facZ9iD8bTzzA%2B5lRqjqj4ZoANXPfMLngC5yFphP4iYxnIQGmv-vZ1fpWqzOpp3klJg0fC0cPn4T7PoiGerlpFO98zrpwNoqa6r9NCtVgFgAA4tgRGyNECo3XH-Ecl9AHOVrlBSqYCu7GGqODHeltG5BnzMpaAABrfMEdqHYBoIKUgdgqyuATCLdG8RIp9SxjfIBeNhpJXwRNbupMPi%2BgDOQ7As4qTt0QO1GQpAqBrlIGZH4XEIhyB6o2H6JVk4CIBkIteGdFR8z7oGcgpB36kE-uuf4lj-BqFLPNWA3xfh%2BBUURbhWDWbHV0j2Yx4C4jVGNuYqypAfy3GnBQDRkQFEXGUWjSuXC1ILxZjgmieDAkEKMA6YyJCG6inwFWMI8BcBuFgDQE%2BEim5eNSbog6%2Bjfa4ICY-dehIHTZ12EgyIVBsh%2BHQAAaUQIgI4biqRYG%2BLQ3ANjz5xG8Wk%2BWy9BEeTaSYx0vcc4UJmVQ0UNBGrNR%2BL6OgiBzakKbkglGVScLaJ4UzPh2C76ZNaUdJ%2B8QHTby2YUQK8jaDQCEH0r0NZ4BPiUeudq0B3AR08apJ6PiMn%2BMJjklQDoX45x6QC-pDxfBNTULhNQRxxSSlhVFXhv1b7AJOg-V57T3mQJzsM0ZM1S6aOFNs0guygzJLSqwK%2B-AHm%2BISkYtZQTbAOmITnHFgQsBBAQMGFJcK0mLD4CCc4NBJhVI2N46EQA)

You could do something like:
```fsharp
let genIndexExpr = genExpr astContext indexExpr

 genExpr astContext identifierExpr
  +> sepOpenLFixed
  +> expressionFitsOnRestOfLine genIndexExpr  (atCurrentColumnIndent genIndexExpr)
  +> sepCloseLFixed
```
 
People shouldn't write multiline indexer expressions in the first place, so I'm not overly bothered by the result of these.

Reply:
Makes sense, I've changed it to trigger on multiple line rather than sequential.

I've had a go with the changelog. I didn't see an example of how duplicates are listed so added them after.

Reply:
Changed to requested style below of listing each issue